script/Ciel/Day 13/13_01_02_CIEL13_7B.txt

```diff
@@ -67,12 +67,12 @@ I couldn't shut my mouth even if I wanted to.
 [sha:04c34d867a4b2f14580f5cd01433a9fce3f4ea2f]{
 -- Page 236, Offset 35939.
 -- 殺したくても。
-Killing myself.
+I couldn't kill myself.
 }
 [sha:2b9128a48be8505a22c50b99a23e55c20bdf24be]{
 -- Page 236, Offset 35940.
 -- 自分を殺したくても、殺せなかった。
-I couldn't kill myself even if I wanted to.
+I couldn't kill myself, even if I wanted to.
```

Comment:
as much as I'd love "I wouldn't be killed even if I killed myself", I think this might be a little bit better